arkouda/series.py

```diff
@@ -486,26 +507,28 @@ def from_return_msg(repMsg):
         return Series(data=vals, index=ind)
 
     @staticmethod
-    def _all_aligned(array):
+    @typechecked
+    def _all_aligned(array: List) -> bool:
         """Is an array of Series indexed aligned?"""
 
         itor = iter(array)
         a1 = next(itor).index
         for a2 in itor:
-            if not a1._check_aligned(a2.index):
+            if a1._check_aligned(a2.index) is False:
                 return False
         return True
 
     @staticmethod
-    def concat(arrays, axis=0, index_labels=None, value_labels=None):
-        """Concatenate in arkouda a list of arkouda Series or grouped arkouda arrays horizontally or vertically.
-
-        If a list of grouped arkouda arrays is passed they are converted to a series.
-        Each grouping is a 2-tuple with the first item being the key(s) and the second being the value.
-
-        If horizontal, each series or grouping must have the same length and the same index.
-        The index of the series is converted to a column in the dataframe.
-        If it is a multi-index,each level is converted to a column.
+    @typechecked
+    def concat(
+        arrays: List, axis: int = 0, index_labels: List[str] = None, value_labels: List[str] = None
+    ) -> Union[arkouda.dataframe.DataFrame, Series]:
```

Comment:
Just want to double check that `index_labels` shouldn't be `Union[List[str], str]`. I don't know for certain but the doc string leads me to believe this could be a single name

Reply:
We spoke offline, it seems `index_labels` should be the same length as `arrays`. We're going to leave it as is for now